src/test/com/github/ast/parser/KParserImplTest.kt

```diff
@@ -4,7 +4,6 @@ import com.google.gson.Gson
 import com.google.gson.JsonObject
 import kastree.ast.psi.Parser
 import mu.KotlinLogging
-import org.jetbrains.kotlin.compilerRunner.KotlinLogger
```

Comment:
@ahinchman1 Not sure why this isn't resolving. Maybe the namespace is incorrect? I'm not familiar with KotlinLogger so not sure. But it looks like you're using `mu.KotlinLogger`, which I'm guessing would be a replacement?